src/store.cc

```diff
@@ -967,46 +961,39 @@ StoreEntry::checkCachable()
         return 0; // avoid rerequesting release below
     }
 
-#if CACHE_ALL_METHODS
-
-    if (mem_obj->method != Http::METHOD_GET) {
-        debugs(20, 2, "StoreEntry::checkCachable: NO: non-GET method");
-        ++store_check_cachable_hist.no.non_get;
```

Comment:
Please remove the now-unused `non_get` counter.

Reply:
Done in commit cdac803b4